var/spack/repos/builtin/packages/py-pygit2/package.py

```diff
@@ -23,17 +25,21 @@ class PyPygit2(PythonPackage):
     depends_on("py-setuptools", type="build")
     # Version must match with libgit2
     # See: https://www.pygit2.org/install.html
-    depends_on("libgit2@1.1.0:1.1", when="@1.4:")
+    depends_on("libgit2@1.6.0:1.6", when="@1.12.0:1.12")
```

Comment:
```suggestion
    depends_on("libgit2@1.6", when="@1.12")
```
Same can be used for other versions